static/panes/tool.js

```diff
@@ -511,6 +515,29 @@ Tool.prototype.onCompileResult = function (id, compiler, result) {
             if (toolResult.sourcechanged && this.editorId) {
                 this.eventHub.emit('newSource', this.editorId, toolResult.newsource);
             }
+            this.artifactBtn.off('click');
+            if (toolResult.artifactGenerated) {
+                this.artifactBtn.removeClass('d-none');
+                this.artifactText.text(`Download ${toolResult.artifactGenerated.title}`);
+                this.artifactBtn.click(
+                    _.bind(function () {
+                        if (toolResult.artifactGenerated.type === 'application/octet-stream') {
+                            fetch('data:application/octet-stream;base64,' + toolResult.artifactGenerated.content)
```

Comment:
Could you maybe write a quick comment on why fetch() has to/is used here over new Blob like below?

Reply:
Is what I added enough for that?